projects/material-addons/src/lib/readonly/readonly-form-field/readonly-form-field.component.ts

```diff
@@ -30,57 +31,43 @@ import { MatFormFieldModule } from '@angular/material/form-field';
 @Component({
   selector: 'mad-readonly-form-field',
   templateUrl: './readonly-form-field.component.html',
-  styleUrls: ['./readonly-form-field.component.css'],
+  styleUrls: ['./readonly-form-field.component.scss'],
   imports: [MatFormFieldModule, NgIf, MatInputModule, FormsModule, NgStyle, NgClass, MatTooltipModule, TextFieldModule, MatIconModule],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ReadOnlyFormFieldComponent implements OnChanges, AfterViewChecked {
-  @ViewChild('contentWrapper', { static: false })
-  originalContent: ElementRef;
-  @Input('useProjectedContent') useProjectedContent: boolean = false;
-  @Input('value') value?: any;
-  @Input('label') label: string;
-  @Input('textAlign') textAlign: 'right' | 'left' = 'left';
-  @Input('formatNumber') formatNumber = false;
-  @Input('decimalPlaces') decimalPlaces = 2;
-  @Input('roundDisplayValue') roundValue = false;
-  @Input('autofillDecimals') autofillDecimals = false;
-  @Input('unit') unit: string | null = null;
-  @Input('unitPosition') unitPosition: 'right' | 'left' = 'left';
-  @Input('errorMessage') errorMessage: string | null = null;
+export class ReadOnlyFormFieldComponent implements OnChanges, AfterViewInit {
+  @Input() useProjectedContent: boolean = false;
+  @Input() value?: any;
+  @Input() label!: string;
```

Comment:
This & other files, same as above:
```suggestion
  @Input({ required: true }) label: string;
```